src/reputation/models.py

```diff
@@ -124,6 +124,12 @@ def set_withdrawal(self, withdrawal_instance):
         self.withdrawal = withdrawal_instance
         self.save()
 
+class Webhook(models.Model):
+    body = JSONField(blank=True)
```

Comment:
Missing `null=True`?

Reply:
The webhook shouldn't be null, if it is then it should error